src/vs/workbench/browser/parts/editor/tabsTitleControl.ts

```diff
@@ -211,6 +214,10 @@ export class TabsTitleControl extends TitleControl {
 			}
 		}));
 
+		// Configuration updates
+		this.toUnbind.push(this.configurationService.onDidUpdateConfiguration(() => this.onConfigurationChanged()));
```

Comment:
I typically prefer to setup listeners from the constructor via `registerListeners` (see in parent class: https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/browser/parts/editor/titleControl.ts#L146). Can we do the same in this class and call the super one?

Reply:
I attempted this like so:

```
	protected registerListeners(): void {
		super.registerListeners();

		// Configuration updates
		this.toUnbind.push(this.configurationService.onDidUpdateConfiguration(() => this.onConfigurationChanged()));
		this.onConfigurationChanged(false);
	}
```

Which results in the following error on startup:
```
Cannot read property 'onDidUpdateConfiguration' of undefined
    at TabsTitleControl.registerListeners (file:///Users/tvald/Documents/Tuple/git/vscode/out/vs/workbench/browser/parts/editor/tabsTitleControl.js:47:57)
```

It appears that `configurationService` isn't yet initialized at this point. I haven't looked into the injection framework, so I don't know why `configurationService` is different from everything else being injected.

Reply:
ok leave it as is and I will have a look after you are done 👍